blocks/identity-block/features/header-account-action/default.jsx

```diff
@@ -48,6 +57,42 @@ const HeaderAccountAction = ({ customFields }) => {
     return null;
   }
 
+  const AccountMenu = () => (
+    <ul className="menu">
+      <PrimaryFont as="li" className="menu-item">Manage Your Account</PrimaryFont>
```

Comment:
All text strings must come from `intl.json` files and not be hard coded into the feature files